src/sdl/sdl_input_backend.cpp

```diff
@@ -6,11 +6,28 @@
 
 #include <iostream>
 
-const char* controller_db =
-#include "sdl/sdl_controller_db.hpp"
+// Split the controller DB to bypass ANSI compatibility string character limit.
```

Comment:
as I mentioned, the vs compiler has a string character limit of 60k bytes if I' m not mistaken. This makes sure that that limit isn't reached. Not sure if this is the best way to do it tho, but it works.